extensions/parquet/table/src/main/java/io/deephaven/parquet/table/metadata/TableInfo.java

```diff
@@ -61,11 +61,11 @@ public final Map<String, ColumnTypeInfo> columnTypeMap() {
     }
 
     /**
-     * @return The Deephaven release version when this metadata format was defined
+     * @return The Deephaven release version when this metadata format was updated
      */
     @Value.Default
     public String version() {
-        return "0.4.0";
+        return "0.31.0";
     }
```

Comment:
I think we made the change assuming it would be Larry's change that will change the metadata.
But you are right Devin, its not obvious that this needs to be changed every time we change the metadata.

@rcaudy  can you please look at this comment once if its okay to change this field's semantics to writeVersion?

Reply:
I think we can safely re-word the semantics of the version field we have, and not introduce a new one.
It might be useful to always reflect the version we wrote with, but I think that requires some automation we don't currently have.

Reply:
@rcaudy : `TableInfo.class.getPackage().getImplementationVersion()`

Reply:
That works? Nice!